proto/contlcycle/contlcycle.proto

```diff
@@ -0,0 +1,20 @@
+syntax = "proto3";
+
+option go_package = "github.com/DataDog/agent-payload/v5/contlcycle";
+
+package datadog.contlcycle;
+
+message EventsPaylaod {
```

Comment:
is it possible to add comments to the `message` and field that will be added to the generated code?

Reply:
it seems possible: https://developers.google.com/protocol-buffers/docs/proto3#adding_comments